lib/iron_bank/schema.rb

```diff
@@ -39,6 +39,9 @@ def self.reset
     end
 
     def self.excluded_fields
+      # When Zuora return InternalServerError we can not extract fields from the a message.
```

Comment:
let's use the notice from here: https://github.com/zendesk/billing/pull/16349/files#r1424247726